src/android/TransitionNotificationReceiver.java

```diff
@@ -0,0 +1,402 @@
+package edu.berkeley.eecs.emission.cordova.transitionnotify;
```

Comment:
Why do you have this? I thought you were going with the approach of having the service generate the notification directly.